package.json

```
@@ -19,10 +19,11 @@
     "eslint-fix": "eslint --ext .js,.mjs,.ts --fix lib/ test/ tools/",
     "generate": "run-s generate:*",
     "generate:imports": "node tools/generate-imports.mjs",
+    "git-check": "node tools/check-git-version.js",
```

```suggestion
    "git-check": "node tools/check-git-version.mjs",
```

Not sure what the correct filename extension is when you run a script in `package.json`.

The file is called `check-git-version.mjs` in the PR, but the script uses `.js` as filename ending.

---

Comitted